src/pages/Archive.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import SubLink from "./SubLink";
+import "../css/link.css";
+
+const Archive = () => {
+  return (
+    <>
+      <div className="wrapper">
+        <div className="slider">
+          <SubLink />
+        </div>
+
+        <div className="slider-content">
+          <h1> Archive PAGE</h1>
+        </div>
+      </div>
+    </>
```

Comment:
Here as well, you don't need a React Fragment, also the text is partially uppercase and partially lowercase, you should fix that